src/visualizers/panels/ExecutionIndex/ExecutionIndexControl.js

```diff
@@ -300,37 +327,28 @@ define([
     };
 
     ExecutionIndexControl.prototype._onUnload = function (id) {
-        var execId = this._lineToExec[id],
+        var execId = this._graphToExec[id],
             abbr;
 
-        if (execId) {  // it is a line
-            delete this._lineToExec[id];
-            for (var k = this._linesForExecution[execId].length; k--;) {
-                if (this._linesForExecution[execId][k] === id) {
-                    this._linesForExecution[execId].splice(k, 1);
-                    break;
-                }
-            }
+        if (execId) {  // it is a graph
+            delete this._graphToExec[id];
+            delete this._graphsForExecution[execId];
         }
-
         if (this.abbrFor[id]) {
             abbr = this.abbrFor[id];
             delete this.abbrFor[id];
             delete this.abbrToId[abbr];
         }
-
         this._widget.removeNode(id);
     };
 
-    ExecutionIndexControl.prototype.isLineDisplayed = function (line) {
+    ExecutionIndexControl.prototype.isGraphDisplayed = function (graph) {
         // lines are only displayed if their execution is checked
-        return this.displayedExecutions[line.execId];
+        return this.displayedExecutions.indexOf(graph.execId) !== -1;
```

Comment:
`.includes(graph.execId)` is a bit more legible, I think

Reply:
`.includes` it is.